### PR TITLE
Adding support for additional docker labels.

### PR DESCRIPTION
**Pull request by FWeinb**
Implemented my request from exoframejs/exoframe#85. Despite the suggestion of adding it as an array of strings I implemented it using an object like it is used in the node api. 

Tests are failing because I added another deployment. Maybe the current test should be integrated into the simple html deploy test. I did not want to change all the tests without first discussing this.

### CODE REVIEW

**Comment by yamalight** [comment]
@FWeinb looks good, but please remove the container after test passes (that should fix the tests)

File: test/deploy.js [hunk 1]
```diff
@@ -484,6 +485,36 @@ module.exports = (server, token) =>
...
+        t.equal(response.statusCode, 200, 'Correct status code');
+
+
+
+        // check docker services
+        const allContainers = await docker.listContainers();
+        const containerInfo = allContainers.find(c => c.Names.includes(deployments[0].Name));
+        t.ok(containerInfo, 'Docker has container');
+        t.equal(containerInfo.Labels['custom.label'], "additional-label", 'Should have label `custom.label=additional-label`');
+
```
Comment (yamalight) [inline test/deploy.js]: Add cleanup, e.g.:
```js
// cleanup
const instance = docker.getContainer(Id);
await instance.stop();
await instance.remove();
```

And it's great 👍

Comment (FWeinb) [reply test/deploy.js]: Thanks. Was missing that bit!

**Comment by yamalight** [comment]
@FWeinb looks good! thanks for the work :)

**Comment by FWeinb** [comment]
No Problem! Thanks for starting this project. Really easy to get setup!

**Pull request merged**